src/Sulu/Bundle/MediaBundle/Tests/Application/Kernel.php

```diff
@@ -20,6 +21,10 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     {
         parent::registerContainerConfiguration($loader);
 
-        $loader->load(__DIR__ . '/config/config.yml');
+        if (SuluKernel::CONTEXT_WEBSITE === $this->getContext()) {
```

Comment:
Why was that necessary?

Reply:
To run MediaStreamControllerTest and MediaWebsiteControllerTest under the correct WebsiteKernel as they test website routes and not admin routes.